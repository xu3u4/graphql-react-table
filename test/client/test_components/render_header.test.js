import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import RenderHeader from 'components/view_issues/render_header';

jest.mock('data');

describe('Render <RenderHeader>', () => {
  const { headers } = require('data'); // eslint-disable-line
  const thead = renderer.create(
    <RenderHeader columns = {headers} />
  );

  it('should render all columns inside <thead>', function() {
    expect(thead).toMatchSnapshot();
  });
});
