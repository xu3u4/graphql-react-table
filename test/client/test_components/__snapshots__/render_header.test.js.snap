// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render <RenderHeader> should render all columns inside <thead> 1`] = `
<thead>
  <tr>
    <th>
      seq
    </th>
    <th>
      Status
    </th>
    <th>
      Category
    </th>
    <th>
      Title
    </th>
    <th>
      Owner
    </th>
    <th>
      Priority
    </th>
    <th>
      Action
    </th>
  </tr>
</thead>
`;
